identity/src/main/java/com/stripe/android/identity/navigation/ErrorFragment.kt

```diff
@@ -82,4 +86,55 @@ internal class ErrorFragment(
             )
         }
     }
+
+    /**
+     * Temporarily convert String route back to a Fragment ID.
+     * TODO(ccen) Remove this method after migrated to Jetpack compose.
+     */
+    private fun String.toFragmentId(): Int {
```

Comment:
Note: because under the hood we're still using `NavController.navigate(destination: Int)`, we added a temporary converting method `String.toFragmentId` to convert the String `destination`  back to Int `fragmentID`. When later migrated to compose, this method can be deleted and we can directly compare `navController.currentDestination?.route == destination`